FlexateWebApi/Controllers/PeopleController.cs

```diff
@@ -149,12 +154,28 @@ public async Task<ActionResult> Patch(int id, CancellationToken cancellationToke
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         public async Task<ActionResult> Delete(int id, CancellationToken cancellationToken)
         {
-            var result = await _peopleService.DeletePerson(id, cancellationToken);
-            if (result)
+            var succeed = await _peopleService.DeletePerson(id, cancellationToken);
+            if (succeed)
             {
                 return NoContent();
             }
             return NotFound();
         }
+
+        [HttpGet("cars,offices/{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult> GetFiltered(CancellationToken cancellationToken, int id)
```

Comment:
also while using cancelation token it may be a good idea to have it as an optional parameter, and normally use default cancelation token. Also when you have params of the method it is better to put cancelation token or paging parameters at the end and have id at the front :)